packages/core/src/index.ts

```diff
@@ -552,7 +552,7 @@ class Hopr extends EventEmitter {
    * @param peer peer to query for, default self
    * @param timeout [optional] custom timeout for DHT query
    */
-  public async getAnnouncedAddresses(peer: PeerId = this.getId(), timeout = 5e3): Promise<Multiaddr[]> {
+  public async getMyAnnouncedAddresses(peer: PeerId = this.getId(), timeout = 5e3): Promise<Multiaddr[]> {
```

Comment:
If already changing the names, this should be called smth like `getAddressesAnnouncedToDHT`. This will clearly distinguish between addresses announced on-chain.